docs/gettingstarted.rst

```diff
@@ -38,31 +38,45 @@ Quickstart
 
        Install `git <https://git-scm.com/>`_.
 
-2. Clone the repository so you have a copy on your host machine. Instructions
-   are `on GitHub <https://github.com/mozilla-services/socorro>`_.
+2. Clone the repository so you have a copy on your host machine.
 
-3. From the root of this repository, run::
+   Instructions for cloning are `on the Socorro page in GitHub
+   <https://github.com/mozilla-services/socorro>`_.
+
+
+3. (*Optional/Advanced*) Set UID and GID for Docker container user.
+
+   If you're on Linux or you want to set the UID/GID of the app user that
+   runs in the Docker containers, run::
+
+     $ make my.env
+
+
```

Comment:
Eh, it's two newlines in most (but not all) other spots in the file. I'll just leave it alone.

Reply:
This is my fault. rst doesn't explicitly require two lines after literal blocks, but I find it's a lot easier to read in text files, so I usually do two lines.

I'm game for fixing everything to one line if you think that's helpful. I'm inclined to write a post-it on my monitor to remind myself to stop the madness but otherwise not do anything here.